automation_tools/__init__.py

```diff
@@ -1144,8 +1144,9 @@ def product_install(distribution, create_vm=False, certificate_url=None,
                 execute(setup_foreman_discovery, host=host)
             if os.environ.get('LIBVIRT_KEY_URL') is not None:
                 execute(setup_libvirt_key, host=host)
-            execute(install_puppet_scap_client, host=host)
-            execute(setup_oscap, host=host)
+            if os.environ.get('SATELLITE_VERSION') == '6.1':
```

Comment:
perhaps we might change the conditional to include all 6.1+ versions (with 6.2 being released soon) and only exclude 6.0.